recipes/improver/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "improver" %}
+{% set version = "0.17.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/metoppv/improver/archive/v{{ version }}.tar.gz
+  sha256: 5d09467169c9d03999758a61b7ac57a00f93c1561d80fe28472b63e211b8320a
+
+build:
+  skip: True  # [win]
+  # Required because cftime 1.0.1 has builds only up to py37
+  skip: True  # [py>=38]
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

---

Yes it can be, the problem was trying resolve the environment for Windows. I think the conflict was due to python abi, for some reason cftime wanted python abi 27, which I think is because [cftime 1.0.1](https://anaconda.org/conda-forge/cftime/files?version=1.0.1) (pinned) has a build variant with Python 2.7. I tried constraining the build variant but I kept getting conflicts. Apologies, I've cleaned up my 25 commits trying to get CIs to pass, so I'm not sure we can see the CI outputs now.
We decided to skip Windows since we don't support it anyway (do not run our package CIs on it). Would you prefer we go with noarch and try to resolve the dependencies?

---

It's fine for the Windows CI to fail with missing dependencies for `noarch: python` packages. After this is merged conda-forge will only ever build the packages on Linux.

---

Thanks @chrisburr, have amended it to noarch.